src/core/middleware-manager.ts

```
@@ -0,0 +1,89 @@
+    
+    if (this.middlewares.length === 0) {
+      this.logger.log('debug', 'No middleware to execute, returning original value');
+      return value;
+    }
+
+    // Create a copy of the middlewares array to avoid issues if middleware is added during execution
+    const middlewares = [...this.middlewares];
+    
+    try {
+      // Simple sequential execution
+      let currentVal = value;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'currentVal' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)